LINUX.md

```diff
@@ -480,6 +480,39 @@ plugins=(gitfast last-working-dir common-aliases zsh-syntax-highlighting history
 
 ## Installing Python (with [`pyenv`](https://github.com/pyenv/pyenv))
 
+### Uninstall `conda`
+
+As we are using `pyenv` to install and manage our Python version, we need to uninstall [`conda`](https://docs.conda.io/projects/conda/en/latest/), another package manager you may have on your machine if you previously installed [Anaconda](https://www.anaconda.com/). Thus, we are preventing any possible Python version issue later.
+
+Check if you have `conda` installed on your machine:
+```bash
+conda list
+```
+If you have `zsh: command not found: conda`, you can **skip** the uninstall of `conda` and jump to the **Install `pyenv`** section.
+
+<details>
+    <summary markdown='span'><code>conda</code> uninstall instructions</summary>
+
+- Install the Anaconda-Clean package from your terminal and run the cleaning
+```bash
+conda install anaconda-clean
+anaconda-clean --yes
+```
+- Remove every Anaconda directories
+```bash
+rm -rf ~/anaconda2
+rm -rf ~/anaconda3
```

Comment:
In order to test the instructions, I've installed Anaconda on my machine [following the conda doc instructions](https://docs.conda.io/projects/conda/en/latest/user-guide/install/linux.html) and then followed the present uninstallation instructions. 

When running the `anaconda-clea --yes` command, a folder got created:

```bash
(base) ➜  Downloads anaconda-clean --yes
Backup directory: /home/dmilon/.anaconda_backup/2022-07-15T105951
```

So I would recommend to add: 
```suggestion
rm -rf ~/anaconda3
rm -rf ~/.anaconda_backup
```